parsec/parsec.c

```diff
@@ -2759,3 +2759,50 @@ void parsec_set_my_execution_stream(parsec_execution_stream_t *es)
 {
     PARSEC_TLS_SET_SPECIFIC(parsec_tls_execution_stream, es);
 }
+
+
+/**
+ * Query the number of devices of the requested type available in the
+ * PaRSEC context.
+ */
```

Comment:
It's not just deviced :)

```suggestion
/**
 * Query the numerical properties of the given
 * PaRSEC context.
 */
```